developer/README.md

```diff
@@ -0,0 +1,3 @@
+# Developer Docs
```

Comment:
Would it just make sense to move `developer.md` into this README?  

I thought it was weird that docs/developer has no README.

Reply:
I see.  This is mostly here for your environment variables?

Reply:
So I think you should make this README a bit more useful.  What are all these folders in developer?  Why should a developer care about this?  Let's try and document this as its a bit confusing without any context.  

You could keep this mention to the developer documentation but what is the use of this folder?  its not obvious to me as I haven't spent very much on the localdev development.

Reply:
Good points. I think what I'll aim to do is also add to that readme an explanation of why the folder exists.

I think all major docs should belong in the docs folder though, so that we can keep it on the website.